HealthcareBase/Service/UsersService/EmployeeService/DoctorService.cs

```diff
@@ -70,5 +70,19 @@ public IEnumerable<DoctorDto> GetDoctorsByDepartment(int departmentId)
                 }
                 );
         }
+
+        public IEnumerable<DoctorDto> GetAllSpecialists()
+        {
+            return doctorRepository.Repository.GetColumnsForMatching(
+                condition: doctor => doctor.DepartmentId != 1,
```

Comment:
I don't understand why doctor is specialist if he is not from department with id 1.

Reply:
Because department ID number 1 is for regular doctor and all others are for specialists

Reply:
It would be better to create at least some constant that will have a meaningful name for that value